app/presenters/result_set_presenter.rb

```diff
@@ -82,6 +110,22 @@ def signup_links
     @signup_links ||= fetch_signup_links
   end
 
+  def next_and_prev_links
+    return unless finder.pagination
+
+    current_page = finder.pagination['current_page']
+    previous_page = current_page - 1 if current_page > 1
+    next_page = current_page + 1 if current_page < finder.pagination['total_pages']
+    pages = {}
+
+    pages[:previous_page] = build_page_link("Previous page", previous_page) if previous_page
+    pages[:next_page] = build_page_link("Next page", next_page) if next_page
+
+    markup = ""
+    markup = view_context.render(formats: %w[html], partial: 'govuk_publishing_components/components/previous_and_next_navigation', locals: pages) if pages
+    markup.squish unless markup.blank?
```

Comment:
`markup` is always going to be a string, isn't it? In which case, `markup.squish` is safe to call even if the string is `""`.

Reply:
I thought so too, but I got errors trying that. I'll check again.

Reply:
Refactored.